torch/nn/functional.py

```diff
@@ -3066,8 +3063,7 @@ def binary_cross_entropy(
     reduce: Optional[bool] = None,
     reduction: str = "mean",
 ) -> Tensor:
-    r"""Function that measures the Binary Cross Entropy between the target and input
-    probabilities.
+    r"""Calculate Binary Cross Entropy between the target and input probabilities.
```

Comment:
Probably better stick to the original "measure" rather than calculate:
```suggestion
    r"""Measure Binary Cross Entropy between the target and input probabilities.
```